incubator/hnc/test/e2e/demo_test.go

```diff
@@ -87,4 +92,37 @@ var _ = Describe("Demo", func() {
 		RunShouldContain(nsService1, 2, "kubectl hns describe", nsTeamA)
 		RunShouldNotContain("my-creds", 2, "kubectl -n", nsService1, "get secrets")
 	})
+
+	It("Should create and delete subnamespaces", func(){
+		MustRun("kubectl create ns", nsOrg)
+		MustRun("kubectl hns create", nsTeamA, "-n", nsOrg)
+		MustRun("kubectl hns create", nsService1, "-n", nsTeamA)
+		MustRun("kubectl hns create", nsService2, "-n", nsTeamA)
+		MustRun("kubectl hns create", nsService3, "-n", nsTeamA)
+		RunShouldContain(nsTeamA+"\n├── "+nsService1+"\n├── "+nsService2+"\n└── "+nsService3, 2, "kubectl hns tree", nsTeamA)
```

Comment:
This might be more clear (and easier to edit) as several lines, eg:
```
expected := "" + // empty string make go fmt happy
	nsTeamA + "\n" +
	"├── " + nsService1 + "\n" +
	"├── " + nsService2 + "\n" +
	"└── " + nsService3
RunShouldContain(expected, 2, "kubectl hns tree", nsTeamA)
```

In case you're wondering, the empty initial string make `go fmt` much happier (try it out on play.golang.org and hit the "format" button to see what `go fmt` does).

Reply:
done